internal/net/listener.go

```diff
@@ -74,6 +74,7 @@ func NewGRPCListenerForPrivate(ctx context.Context, bindingAddr string, s Servic
 	drand.RegisterPublicServer(grpcServer, s)
 	drand.RegisterProtocolServer(grpcServer, s)
 	pdkg.RegisterDKGControlServer(grpcServer, s)
```

Comment:
Are you sure you want to leave that one here?

Reply:
jesus my brain is not here